content/guide/install-uninstall.md

```diff
@@ -323,4 +323,80 @@ To uninstall Quire:
 
 ## Troubleshooting
 
-TK
+**Errors installing Prince**
+
+When installing versions of Prince newer than 12.5 you will need to follow these steps:
+
+1. Copy and paste the following code:
+
+    ```text
+    cd Downloads/prince-12.5-macosx
+    sudo ./install.sh
+    ```
+
+The key line is **prince-XX.X-macos(x)**
+
+2. Go to the Downloads folder to make sure the Prince zip file is there.
+
+3. Make sure XX.X matches the current version of Prince that you are using.
+
+4. The line may end in macos or macosx. Make sure the it matches exactly what the program looks like in the download folder.
+
+
+**Downloading the Quire CLI to your computer from GitHub through the Terminal**
+
+If you have two-factor authentication set-up, you may need to create a personal access token in GitHub to get Quire CLI to download properly.
```

Comment:
`two-factor authentication` would be a good term to add to the glossary and use the q-glossary shortcode for it here